patroni/config.py

```diff
@@ -614,3 +615,25 @@ def get_global_config(self, cluster: Union[Cluster, None]) -> GlobalConfig:
         :returns: :class:`GlobalConfig` object
         """
         return get_global_config(cluster, self._dynamic_configuration)
+
+    def _validate_failover_tags(self):
+        """Checks nofailover/failover_priority config and warns user if it's contradictory"""
+        nofailover_tag = self.get('tags', {}).get('nofailover', None)
+        failover_priority_tag = self.get('tags', {}).get('failover_priority', None)
+        # First handle case where `failover_priority` is str, not int
+        if isinstance(failover_priority_tag, str):
+            self['tags']['failover_priority'] = parse_int(failover_priority_tag)
+            failover_priority_tag = self.get('tags', {}).get('failover_priority', None)
+        # Next validate semantics between `failover_priority` and `nofailover`
+        if nofailover_tag is not None and failover_priority_tag is not None:
+            if (
+                nofailover_tag is True and failover_priority_tag > 0
+                or nofailover_tag is False and failover_priority_tag <= 0
+            ):
+                logger.warning(
+                    'Conflicting configuration between nofailover: %s and failover_priority: %s.'
+                    + ' Defaulting to nofailover: %s',
```

Comment:
You can use implicit string concatenation inside brackets
```suggestion
                    'Conflicting configuration between nofailover: %s and failover_priority: %s.'
                    ' Defaulting to nofailover: %s',
```